src/style.css

```diff
@@ -0,0 +1,31 @@
+* {
+  margin: 0;
+}
+
+.stars--0,
```

Comment:
Create a common starts class, where there will be common styles, and attach it to each block